jerry-core/ecma/base/ecma-globals.h

```diff
@@ -994,21 +1014,28 @@ typedef double ecma_number_t;
  */
 #define ECMA_NUMBER_MINUS_ONE ((ecma_number_t) -1)
 
+#if CONFIG_ECMA_NUMBER_TYPE == CONFIG_ECMA_NUMBER_FLOAT32
 /**
- * Minimum positive and maximum value of ecma-number
+ * Number.MIN_VALUE (i.e., the smallest positive value of ecma-number)
+ *
+ * See also: ECMA_262 v5, 15.7.3.3
```

Comment:
I don't really like these comment duplications. Is there a better way to do this?

Reply:
There are no easy ways and/or this is the way that's compatible with the rest of ecma-globals.h. (E.g., see the documentation of `ecma_number_t`, `ECMA_NUMBER_MAX_DIGITS`, `ECMA_NUMBER_SIGN_WIDTH`, `ECMA_NUMBER_BIASED_EXP_WIDTH`, and `ECMA_NUMBER_FRACTION_WIDTH`.) I've followed that approach here, too.

FYI, there may be one way to reduce duplication by using `@def MACRONAME(args)` doxygen syntax to break out documentation from before the macro definitions, but that might necessitate a bigger rewrite throughout the code base (to apply that approach everywhere where macros (or typedefs, or functions) are conditionally defined multiple times). Moreover, that approach should be discussed further as it splits the documentation from the documented entity, making maintenance somewhat harder. So, I'd have that separate from this PR.

(To restate: this PR tries to fix the problem in the Doxyfile with the least friction, fixing only the fallout that can be easily fixed, leaving the rest for follow-up.)

Reply:
You can use `@copydoc` or groups to avoid duplication in comments. Maybe my previous PR can help: https://github.com/jerryscript-project/jerryscript/pull/2324

Reply:
But you cannot copy documentation when an entity is defined multiple times because of conditional compilation, can you? I guess you cannot copydoc self.

```c
#if COND
/** documentation of X */
#define X
/** documentation of Y */
#define Y
#else
/** @copydoc X */   // <-- THIS SHOULDN'T WORK
#define X
/** @copydoc Y */   // <-- THIS SHOULDN'T WORK
#define Y
#endif
```

Reply:
IHMO the default macro conditions should be documented. The Doxygen docs will be generated with the default config. BTW you can use a group comment before the `#if` which should work independently from the configuration. Although this can be useful in simple cases only (like your example in the prev. comment).